src/main/resources/uk/ac/ebi/ena/sra/schema/SRA.analysis.xsd

```diff
@@ -10,7 +10,7 @@
   ~ specific language governing permissions and limitations under the License.
   -->
 
-<!-- version:1.8.0 -->
+<!-- version:1.9.0 -->
```

Comment:
- The gradle build version is 1.9.1 but the version in XDSs is 1.9.0. Perhaps we don't need a version in the XSDs at all.